pisek/tests/kasiopea_test_suite.py

```diff
@@ -194,6 +194,10 @@ def runTest(self):
         if self.generator.cache_used:
             message = "\n  Generátor se nezměnil, používám vstupy vygenerované v předchozím běhu."
             print(termcolor.colored(message, color="cyan"))
+        else:
+            # Remove old solution outputs. The invalidated cache would not be used either
+            # way, but this way we get rid of possible junk
+            util.clean_data_dir(self.task_config, leave_inputs=True)
```

Comment:
ditto

Reply:
Tady to je v pohodě, protože s vygenerovanými daty nic nědeláme, generujeme online. Jde nám jen o ten generátor.